qiskit/providers/ibmq/api_v2/exceptions.py

```diff
@@ -47,3 +47,8 @@ class WebsocketAuthenticationError(WebsocketError):
 class WebsocketTimeoutError(WebsocketError):
     """Timeout during websocket communication."""
     pass
+
+
+class UnlicensedError(ApiError):
```

Comment:
Can you rename a bit, for clarity, to `AuthenticationLicenseError` or similar? Paving the way towards other `AuthenticationError`, and in similarity to the existing `WebsocketAuthenticationError`.